testing/ssh-honeypot/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="ssh-honeypot"
+pkgver="0_git20190717"
+_commit=b7baa9a6963e723ee0779a6cef0447739870ace3
+pkgrel=0
+pkgdesc="Fake sshd that logs ip addresses, usernames, and passwords."
+url="https://github.com/droberson/ssh-honeypot"
+arch="all"
+license="MIT"
+depends="openssh-keygen"
+makedepends="json-c0.12-dev libssh-dev"
+options="!check" # no testsuite and daemon returns 1 even on help command
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.zip::https://github.com/droberson/ssh-honeypot/archive/$_commit.zip
+	$pkgname.initd
+	$pkgname.confd"
+builddir="$srcdir/$pkgname-$_commit"
+
+build() {
+	make CC="gcc" CFLAGS="$CFLAGS -fPIC"
```

Comment:
No need to set `CC` now, that should be set to `CC` by default.

Reply:
I have to because of CC=clang in the Makefile.

Reply:
Sorry about that, you're right. Had thought our env variable would take precedence here (since we export `CC`)